### ISSUE TITLE
[Question] is HTML overlay possible?

### ISSUE THREAD

**Issue by gauravniltech** [bbc/brave#65]
I want to HTML overlay with CSS animations and all. I am able to achieve static overlays by converting the HTML into an image and merging it with the video frames using OpenCV but looking for something like the OBS studio browser overlay feature.

**Comment by moschopsuk** [comment]
Yes this should be possible when we first built this we built a very basic html render Gstreamer element.

Since then more Gstreamer HTML render elements have been created, i urge you to use these as they are in active development.

(chrome) https://github.com/centricular/gstcefsrc
(firefox) https://github.com/servo/servo/tree/master/ports/gstplugin
(Webkit) https://gstreamer.freedesktop.org/documentation/wpe/index.html?gi-language=c
